### PR TITLE
DPIRD science (Independent publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to COVID delays, unanticipated team leaves, and high volume of connector submissions. 

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
image
image
image

- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/DPIRD Science/Readme.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Returns the potential yield values as calculated using the French & Schultz potential yield model. 
+#### The potential yield endpoint uses seasonal rainfall and decile finishes, calculated from historical data, to calculate the maximum wheat yield possible in the absence of any other constraints.
+* Potential yield can be used as a tool in the seasonal decision making process.
+* Estimates of potential wheat yield are obtained using the French & Schultz potential yield model:
+* Yield (tonnes/ha) = WUE * (stored soil water + growing season rainfall - evaporation)
+* WUE is the abbreviation for water use efficiency and growing season rainfall is usually calculated from April to October, inclusive.
+* Stored soil water at the start of the growing season is estimated as one third of summer rainfall.
+* More information on the model available at the [Potential Yield Tool on the DPIRD website](https://www.agric.wa.gov.au/climate-weather/potential-yield-tool)
+
+### Get Soil Wster
```
Comment (v-eisljami) [inline independent-publisher-connectors/DPIRD Science/Readme.md:48]: Readme must be free from grammar and spelling errors.

File: independent-publisher-connectors/DPIRD Science/Readme.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+You can see what information is available from the API, and how you can filter and search for specific information, by exploring the [interactive home page](https://www.agric.wa.gov.au/science-api-20).
+
+## Prerequisites
+You will need the following to proceed
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* The Power platform CLI tools
+* An [API key](https://www.agric.wa.gov.au/form/dpird-api-registration) from DPIRD in accordance with [terms and conditions](https://www.agric.wa.gov.au/apis/api-terms-and-conditions)
+
+## Authentication
```
Comment (v-eisljami) [inline independent-publisher-connectors/DPIRD Science/Readme.md:17]: There is no Obtaining credentials section which is a required section in the readme file. You can always refer to this [template ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)for more information on the readme file.

File: independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1223 @@
...
+              "loamy-earth",
+              "clay"
+            ]
+          },
+          {
+            "name": "faoInitialisationDays",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Number of days in crop initialisation period, used in estimation of crop evapotranspiration.",
```
Comment (v-eisljami) [inline independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json:399]: All descriptions, summaries, titles, etc. must be free from grammar and spelling errors. Please review them.

Comment (paulculmsee) [reply independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json:399]: Line 396 is correct as the source API documentation at DPIRD: 

image

Comment (paulculmsee) [reply independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json:399]: Nope, that one is also correct too. What I will do is run through the spell checker but flag if this is as per the original API. Most of these descriptions are cut/paste from source so a lot of the errors are at source.

Comment (paulculmsee) [reply independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json:399]: Fixed as suggested in various spots

**Review by v-eisljami — APPROVED** [review]

**Comment by v-eisljami** [comment]
[[certify-connector]]

**Review by raghavthind2005 — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hi @paulculmsee, I hope you are doing well. I want to follow up with you and let you know that your connector is in production deployment. this process will complete within 4-6 weeks.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @paulculmsee Congratulations! The deployment of your connector and documentation page is now complete. We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**